src/Compilers/VisualBasic/Portable/Symbols/UnboundGenericType.vb

```diff
@@ -326,7 +326,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return SpecializedCollections.EmptyEnumerable(Of PropertySymbol)()
         End Function
 
-        Private NotInheritable Class ConstructedSymbol
+        Friend NotInheritable Class ConstructedSymbol
```

Comment:
>Friend [](start = 8, length = 6)

It is not clear to me why accessibility is changed here.

Reply:
It's for type check in test:
```VB
Assert.IsType(Of UnboundGenericType.ConstructedSymbol)(u_c3)
```

---
In reply to: [164603526](https://github.com/dotnet/roslyn/pull/24336#discussion_r164603526) [](ancestors = 164603526)

Reply:
I'd rather keep this type private, we can assert type's name instead. Same for the other one

---
In reply to: [164604015](https://github.com/dotnet/roslyn/pull/24336#discussion_r164604015) [](ancestors = 164604015,164603526)